src/components/Checkbox.tsx

```diff
@@ -20,6 +23,8 @@ export interface Props {
   label: string;
   name: string;
   value?: string;
+  disabled?: boolean;
+  checked?: boolean;
```

Comment:
I think we can remove this property because it isn't used

Reply:
I think `disabled` is still being used, but `checked` can be removed